UnRen.bat

```diff
@@ -1,115 +1,114 @@
+:: Configuration:
+::   Set a Quick Save and Quick Load hotkey - http://www.pygame.org/docs/ref/key.html
+:: --------------------------------------------------------------------------------
 set "quicksavekey=K_F5"
 set "quickloadkey=K_F9"
-REM --------------------------------------------------------------------------------
-REM !! END CONFIG !!
-REM --------------------------------------------------------------------------------
-REM The following variables are Base64 encoded strings for unrpyc and rpatool
-REM Due to batch limitations on variable lengths, they need to be split into
-REM multiple variables, and joined later using powershell.
-REM --------------------------------------------------------------------------------
-REM unrpyc by CensoredUsername
-REM   https://github.com/CensoredUsername/unrpyc
-REM Edited to remove multiprocessing and adjust output spacing [UNRPYC//SHA] [UNRPYC//DATE]
-REM   https://github.com/F95Sam/unrpyc
-REM --------------------------------------------------------------------------------
-REM set unrpyccab01=
+:: --------------------------------------------------------------------------------
+:: !! END CONFIG !!
+:: --------------------------------------------------------------------------------
+:: The following variables are Base64 encoded strings for unrpyc and rpatool
+:: Due to batch limitations on variable lengths, they need to be split into
+:: multiple variables, and joined later using powershell.
+:: --------------------------------------------------------------------------------
+:: unrpyc by CensoredUsername
+::   https://github.com/CensoredUsername/unrpyc
+:: Edited to ::ove multiprocessing and adjust output spacing [UNRPYC//SHA] [UNRPYC//DATE]
+::   https://github.com/F95Sam/unrpyc
+:: --------------------------------------------------------------------------------
+:: set unrpyccab01=
 [UNRPYC//BASE]
-REM --------------------------------------------------------------------------------
-REM rpatool by Shizmob [RPATOOL//SHA] [RPATOOL//DATE]
-REM   https://github.com/Shizmob/rpatool
-REM --------------------------------------------------------------------------------
-REM set rpatool01=
+:: --------------------------------------------------------------------------------
+:: rpatool by Shizmob [RPATOOL//SHA] [RPATOOL//DATE]
+::   https://github.com/Shizmob/rpatool
+:: --------------------------------------------------------------------------------
+:: set rpatool01=
 [RPATOOL//BASE]
-REM --------------------------------------------------------------------------------
-REM !! DO NOT EDIT BELOW THIS LINE !!
-REM --------------------------------------------------------------------------------
-set "version=0.10.0-dev ([DEV//BUILD_DATE])"
+:: --------------------------------------------------------------------------------
+:: !! DO NOT EDIT BELOW THIS LINE !!
+:: --------------------------------------------------------------------------------
+set "version=0.11.0-dev ([DEV//BUILD_DATE])"

```

Comment:
should be kept as 0.9.0-dev, as 0.9.0 hasn't been released yet, agreed with semantic versioning change

Reply:
Agreed. The bump was mostly to point out "there where changes".
Switched version back to '0.9.0-dev'